jenkins/bootstrap.py

```diff
@@ -215,7 +215,15 @@ def checkout(call, repo, branch, pull, ssh=False):
         refs, checkouts = branch_ref(branch)
 
     git = 'git'
-    call([git, 'init', repo])
+    if git_cache:
```

Comment:
Personally I would like to see a checkout test for this instead of (or in addition to) the bootstrap test.